internal/io/csv.go

```diff
@@ -151,6 +162,48 @@ func resizeColBytes(bytes [][]byte, currentRowCount, sizeHint int) {
 	}
 }
 
+// Handle DuplicateColumnNames
+func AddColumnIndexToDuplicateColumns(headers []string) []string {
+	headersMap := make(map[string]int)
+	// Loop through all columsn and find the duplicates.
```

Comment:
Consider rewriting this comment to better reflect why this is done or rename `headersMap` to something more telling and remove comment.